format-parquet/pom.xml

```diff
@@ -79,6 +79,10 @@
       <artifactId>jackson-mapper-asl</artifactId>
       <version>1.9.13</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Dont think there is need to add this dependency explicitly.

Reply:
I do see this dependency explicitly specified in other plugin packages like (database-plugins, http-plugins, core-plugins) so its best to follow the same pattern.

Reply:
sounds good